app_dart/test/service/scheduler_test.dart

```diff
@@ -728,9 +803,11 @@ targets:
             'check_suite': <String, dynamic>{'id': 2},
           });
         });
-        final Map<String, dynamic> checkRunEventJson = jsonDecode(checkRunString) as Map<String, dynamic>;
+        final Map<String, dynamic> checkRunEventJson =
+            jsonDecode(checkRunString) as Map<String, dynamic>;
         checkRunEventJson['check_run']['name'] = Scheduler.kCiYamlCheckName;
-        final cocoon_checks.CheckRunEvent checkRunEvent = cocoon_checks.CheckRunEvent.fromJson(checkRunEventJson);
+        final cocoon_checks.CheckRunEvent checkRunEvent =
+            cocoon_checks.CheckRunEvent.fromJson(checkRunEventJson);
```

Comment:
Did you do a 80-column dartfmt?

Reply:
I didn't format correctly. Fixed.

We should totally have a commit check before uploading to save resources.